### PR TITLE
coded pundit for Life

### PR DESCRIPTION
**Pull request by scallywag89**
added pundit gem
coded along with the lecture

### CODE REVIEW

File: app/controllers/lives_controller.rb [hunk 1]
```diff
@@ -22,23 +26,25 @@ def create
...
-    @life = Life.find(params[:id])
   end
 
   def update
-    @life = Life.find(params[:id])
     @life.update(params[:life])
     redirect_to life_path(@life)
   end
 
   def destroy
```
Comment (ninjacazaam) [inline app/controllers/lives_controller.rb:36]: Does this need authorisation also?

Comment (ninjacazaam) [reply app/controllers/lives_controller.rb:36]: It is authorised im det_life, disregard previous comment

Comment (scallywag89) [reply app/controllers/lives_controller.rb:36]: think it should all be covered off in the private set_life method

**Review by ninjacazaam — APPROVED** [review]
Yep looks gwd

**Pull request merged**